packages/manager/modules/octavia-load-balancer/src/components/instances-table/instances-table.html

```diff
@@ -0,0 +1,140 @@
+<oui-button
+    variant="secondary"
+    icon-left="oui-icon-add"
+    disabled="$ctrl.disabled"
+    class="mb-2"
+    data-on-click="$ctrl.addListener()"
+>
+    <span data-translate="octavia_load_balancer_add_listener"></span>
+</oui-button>
+<table class="oui-table">
+    <thead>
+        <tr>
+            <th class="oui-table__header">
+                <span
+                    data-translate="octavia_load_balancer_instances_table_listener"
+                ></span>
+            </th>
+            <th class="oui-table__header"></th>
+            <th class="oui-table__header">
+                <span
+                    data-translate="octavia_load_balancer_instances_table_pools"
+                ></span>
+                <span class="oui-icon oui-icon-help" oui-tooltip="TEST"></span>
```

Comment:
oui-tooltip="TEST" ? Maybe you have the right text ?

Reply:
Ah yes you right I don't have this text I have to ask the requester